packages/xpup/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ericchiang/xpup
+TERMUX_PKG_DESCRIPTION="command line tool for processing XML"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.0.0+20150709172731.8b964504de6f-1
```

Comment:
This is not valid version. If you don't know what to put there, use commit date.